tests/test_pipelines.py

```diff
@@ -63,6 +63,26 @@ def test_duplicates_pipeline_querystring():
         dup_pl.process_item(item4, spider)
 
 
+def test_duplicates_pipeline_courseware_start():
+    dup_pl = DuplicatesPipeline()
+    spider = object()
+    item1 = {"url": "https://courses.edx.org/courses/foo/courseware/bar/baz/"}
+    processed1 = dup_pl.process_item(item1, spider)
+    assert item1 == processed1
+
+    item2 = {"url": "https://courses.edx.org/courses/foo/courseware/bar/baz/1"}
```

Comment:
Up to you, but I wonder if you want some additional testcases here, e.g., `courseware/bar/baz/2` or `courseware/bar/1`.

That's the only comment from my perspective.